components/utility/nostr-helper-functions.ts

```diff
@@ -347,7 +347,9 @@ export const getLocalStorageData = (): LocalStorageInterface => {
     if (
       mints === null ||
       mints[0] ===
-        "https://legend.lnbits.com/cashu/api/v1/4gr9Xcmz3XEkUNwiBiQGoC"
+        "https://legend.lnbits.com/cashu/api/v1/4gr9Xcmz3XEkUNwiBiQGoC" ||
+      mints[0] !==
+        "https://legend.lnbits.com/cashu/api/v1/AptDNABNBXv8gpuywhx6NV"
```

Comment:
Im confused on this logic here. 
If item in mints is this or it isn't this. 

why not just 
mints === null || mints[0] !== "https://legend.lnbits.com/cashu/api/v1/AptDNABNBXv8gpuywhx6NV"